README.md

```diff
@@ -1,2 +1,46 @@
-# website
+# React.Indy website
+
 The website for React.Indy - an Indianapolis-based group for anyone interested in React, React Native, React frameworks (like Next.js), JavaScript, TypeScript, or front-end engineering.
+
+## 1. Contributing
+
+The contributing process documentation is still in-progress. The numbered list below has the basics for the meantime. If you're unsure or stuck about a particular step, first see the ["Related resources"](#1b-related-resources) section directly after the list. If you still need help or reassurance, just ask! :)
+
+### 1a. The general flow
+
+After you've created a fork of this repository under your GitHub account, and cloned your fork to your machine, the contribution flow looks like:
```

Comment:
Contributing via forking the repo sounds good to me! That'd certainly be less to manage on your end if anyone would like to open a PR.